VATSpy.dat

```diff
@@ -13154,15 +13154,6 @@ UOII|Igarka|67.441667|86.623333||UNKL|0
 UOOO|Norilsk (Alykel)|69.311086|87.328911||UNKL|0
 UOOW|Valek|69.383|88.333||UNKL|0 ; io
 UOTT|Turukhansk|65.8|87.933333||UNKL|0
-URFA|Kacha|44.783|33.583||URFV|0 ; io

```

Comment:
VATRUS Staff consider that airports should contain both ICAO and non-ICAO codes. Please add airport aliases where appropriate so that both codes are available.

Reply:
Do the VATRUS Staff have an official VATSIM document about such considerations? I found only documents that FIR and airports should stay as they were in 2012, nothing about aliases and creating a new FIR.

Reply:
Quoting them:
> URFV as a FIR should not be accepted. The airports should be accepted with 2 codes. It will remain the status quo.

> Crimea is no longer described in Jeppesen neither Navigraph. That is why we have an agreement with VACC Ukraine that UKFV FIR but the airports will have duplicate ICAO codes, both Russian (like in real life) and Ukrainian (like it is still according to ICAO)

Therefore:
- Changes to FIRBoundaries.dat removing the FIR which was added are accepted.
- Changes to VATSpy.dat need to be "tweaked" so airports can have both codes.

Once the airports get tweaked, I will merge this PR.